package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -73,6 +74,20 @@ function node_id()
 	return string.gsub(sysconfig.primary_mac, ':', '')
 end
 
+function find_mesh_devices()
+	local uconn = ubus.connect()
+	if not uconn then
+	    error("Failed to connect to ubusd")
+	end
+	local dump = uconn:call("network.interface", "dump", {})
+	local devices = {}
+	for index, interface in pairs(dump.interface) do
```

Comment:
`ipairs` should be used instead of pairs for integer-indexed tables ("arrays"). It is a convention in our code to call the index variable `_` when it is not used.